frontend/packages/noobaa-storage-plugin/src/components/data-consumption-card/data-consumption-card-dropdown.tsx

```diff
@@ -37,7 +37,7 @@ export const DataConsumptionDropdown: React.FC<DataConsumptionDropdownProps> = (
     </DropdownItem>,
   ];
 
-  const kpiDropdownItems = [
+  const kpiDropdownItemsProviders = [
```

Comment:
what about `providersKpiDropdownItems` and `accountKpiDropdownItems` ?

Reply:
ack